backend/.env.sample

```
@@ -2,3 +2,6 @@
 DATABASE_URL=postgres://permaplant:permaplant@127.0.0.1/permaplant
 BIND_ADDRESS_HOST=127.0.0.1
 BIND_ADDRESS_PORT=8080
+
+# OAuth2 (other URLs will be fetched from this URL)
+OAUTH2_ISSUER_URI=http://localhost:8081/realms/PermaplanT/.well-known/openid-configuration
```

We will call it auth.permaplant.net

```suggestion
AUTH_ISSUER_URI=http://localhost:8081/realms/PermaplanT/.well-known/openid-configuration
```